index.html

```diff
@@ -242,8 +242,45 @@ <h1 class="headline-h1">About Me</h1>
         </ul>
       </li>
     </ul>
-  </div>                              
+  </div>
+  
+  <form id="contact-form" action="https://formspree.io/f/xdobgedl" method="POST">
```

Comment:
Addressed ✔ Form is working as expected! Validations issues were solved 🔥

![image](https://user-images.githubusercontent.com/15807343/154092283-ca47bbc4-b6c1-4f85-b5e6-1fbb58c253cb.png)